.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
+      "csstree/validator": true
+    },
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
+  }
```

Comment:
- [ ] Kindly add this file into `.gitignore` This file is to test the linters locally and is not necessary to run the project. Kindly fix this minor issue. 

😄 👍